### PR TITLE
Rename csi to driver

### LINKED ISSUE #75: `make build` references non-existent target

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
Renamed provisioner-csi to csi-driver

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #75 

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/operator_test.go [hunk 1]
```diff
@@ -179,7 +179,6 @@ func TestNodeSelector(t *testing.T) {
 
 	cr, err := hppClient.HostpathprovisionerV1beta1().HostPathProvisioners().Get(context.TODO(), "hostpath-provisioner", metav1.GetOptions{})
 	Expect(err).ToNot(HaveOccurred())
-	Expect(cr.Spec.DisableCsi).To(BeTrue())
```
Comment (aglitke) [inline tests/operator_test.go:182]: How is this related?

Comment (awels) [reply tests/operator_test.go]: Its not, reverted.
